docs/explanations/glossary.md

```diff
@@ -12,6 +12,18 @@ The object representation of the current state of a block in post content. When
 
 The abstract term used to describe units of markup that, composed together, form the content or layout of a webpage. The idea combines concepts of what in WordPress today we achieve with shortcodes, custom HTML, and embed discovery into a single consistent API and user experience.
 
+## Block Styles
+
+The CSS styles that are part of the block, either via its stylesheet or via the block markup itself. For example, a class attached to the block markup is considered block styles.
+
+Compare to <a href="#global-styles">Global Styles</a>. In contraposition to "Global Styles", block styles are sometimes referred to as "Local Styles".
```

Comment:
> Local Styles

Would it be useful to add an entry in this very glossary like so:

```markdown
## Local Styles

See <a href="#block-styles">Block Styles</a>.
```

Reply:
Oh, missed this comment, sorry. https://github.com/WordPress/gutenberg/pull/38083